src/components/mobile/SettingsPage.js

```diff
@@ -16,10 +18,39 @@ import { Slider } from '../ui/Slider'
 import { Tag, TagList } from '../ui/Tag'
 
 const SettingsPage = () => {
+  // TODO: Move form components and type select logic to separate Form page
   const [currentStep, setCurrentStep] = useState(2)
+  const [typeOptions, setTypeOptions] = useState([])
+
+  useEffect(() => {
+    async function fetchTypes() {
+      const types = await getTypes()
+      const options = types.map((t) => ({
+        value: t.id,
+        label: t.name,
+      }))
+      setTypeOptions(options)
+    }
+    fetchTypes()
+  }, [])
+
+  const handleTypeSelect = (types) => {
+    const typeIds = types.map((t) => t.value)
+    console.log('Selected type IDs: ', typeIds)
```

Comment:
Remove printing before merging?

Reply:
We can keep this here -- this kitchen page sink will eventually be removed.